demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1713,15 +1713,21 @@ def is_valid_integration_file_path(self) -> bool:
 
     @error_codes("IN137")
     def is_valid_py_file_names(self):
-        # Gets the all integration .py files from the integration folder.
+        # Files that will be excluded from the check.
         excluded_files = [
             "demistomock.py",
             "conftest.py",
             "CommonServerPython.py",
             "CommonServerUserPython.py",
             ".vulture_whitelist.py",
-            "MicrosoftApiModule.py",  # won't affect the actual API module since it's a script not an integration.
         ]
+
+        # Files that will be excluded from the check if they end with the given suffix (str.endswith).
+        excluded_file_suffix = [
+            "ApiModule.py",
```

Comment:
Let's add the deleted comment back here
# won't affect the actual API module since it's a script not an integration.

Reply:
```suggestion
            "ApiModule.py",  # won't affect the actual API module since it's a script not an integration.
```